File: specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/examples-js/DefaultSecurityRuleGet.js
const { NetworkManagementClient } = require("@azure/arm-network");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Get the specified default network security rule.
 *
 * @summary Get the specified default network security rule.
 * x-ms-original-file: specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/examples/DefaultSecurityRuleGet.json
 */
async function defaultSecurityRuleGet() {
  const subscriptionId = "subid";
  const resourceGroupName = "testrg";
  const networkSecurityGroupName = "nsg1";
  const defaultSecurityRuleName = "AllowVnetInBound";
  const credential = new DefaultAzureCredential();
  const client = new NetworkManagementClient(credential, subscriptionId);
  const result = await client.defaultSecurityRules.get(
    resourceGroupName,
    networkSecurityGroupName,
    defaultSecurityRuleName
  );
  console.log(result);
}

defaultSecurityRuleGet().catch(console.error);

File: specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples-java/DefaultSecurityRuleGet.java

/**
 * Samples for DefaultSecurityRules Get.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/DefaultSecurityRuleGet.json
     */
    /**
     * Sample code: DefaultSecurityRuleGet.
     * 
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void defaultSecurityRuleGet(com.azure.resourcemanager.AzureResourceManager azure) {
        azure.networks().manager().serviceClient().getDefaultSecurityRules().getWithResponse("testrg", "nsg1",
            "AllowVnetInBound", com.azure.core.util.Context.NONE);
    }
}
